### DISCUSSION TITLE
Cannot chat - Set your Key in the Header menu to chat

### DISCUSSION THREAD

**Discussion by asdf23** [LibreChat-AI/docs#76] [Q&A]
I cannot use the website to chat.
Steps to reproduce:

Navigate to:
https://librechat-librechat.hf.space/c/new

Sign in (I used google's link)

I cannot type in the chat field I see the text "Set your Key in the Header menu to chat"

**Comment by fuegovic** [comment]
You do need to provide your own API keys to use our demo.
